pkgs/applications/blockchains/ledger-live-desktop/default.nix

```
@@ -2,11 +2,11 @@
 
 let
   pname = "ledger-live-desktop";
-  version = "2.45.0";
+  version = "2.45.1";
 
   src = fetchurl {
     url = "https://download.live.ledger.com/${pname}-${version}-linux-x86_64.AppImage";
-    hash = "sha256-jw4ocBtyxhPhI2GnhL9tbduY4iIQK53vUHB64qSGXKI=";
+    hash = "sha256-294a7b65067e4c78e2a0e49edc0e34663fb93ad796c44bfe76749b4d433ca8c1=";
```

SRI hashes should be base64 encoded so this is not a valid hash :-)

```suggestion
    hash = "sha256-KUp7ZQZ+THjioOSe3A40Zj+5OteWxEv+dnSbTUM8qME=";
```

---

how can i make sure i get the base64 encoded has, so i know for next time

---

++ i will use the correct naming next time, Apologies for the inconvenience.

I am new to this and just trying to help and learn at the same time

---

I assume you have a local nixpkgs clone and Nix 2.4+, you can just try to update the `version` and modify a few letters from the `hash` and try to build `nix-build -A ledger-live-desktop`, it should fail but it should also gives you the correct SRI hash, then you can try to use that hash and run the build again to actually test the update.

P.s.: Actually if you write `hash = "sha256:294a7b65067e4c78e2a0e49edc0e34663fb93ad796c44bfe76749b4d433ca8c1";` the build should succeed as well since SRI hashes are not really technically required so far, but I am not sure whether other comitters are happy about that.